### ISSUE TITLE
Client and server crashing when placing Better Nether signs

### ISSUE THREAD

**Issue by Greg-J** [paulevsGitch/BetterNether#150]
EDIT: Client and server crash when placing better nether signs

Tried to place a sign (one of the red ones) on a hopper (shift+click) and crashed.  Nobody can join the server now without getting this crash. Restarted the server and client, still no dice.

```
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 2020-07-23 14:05:52 PDT
Description: Rendering Block Entity

java.lang.NullPointerException
    at Not Enough Crashes deobfuscated stack trace.(1.16.1+build.21)
    at net.minecraft.client.render.VertexConsumerProvider$Immediate.getBuffer(VertexConsumerProvider:37)
    at paulevs.betternether.blockentities.render.BNSignBlockEntityRenderer.getConsumer(BNSignBlockEntityRenderer.java:125)
    at paulevs.betternether.blockentities.render.BNSignBlockEntityRenderer.render(BNSignBlockEntityRenderer.java:68)
    at paulevs.betternether.blockentities.render.BNSignBlockEntityRenderer.render(BNSignBlockEntityRenderer.java:35)
    at net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher.render(BlockEntityRenderDispatcher:107)
    at net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher.method_23081(BlockEntityRenderDispatcher:96)
    at net.minecraft.class_824$$Lambda$7161/413246557.run(Unknown Source)
    at net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher.runReported(BlockEntityRenderDispatcher:128)
    at net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher.render(BlockEntityRenderDispatcher:96)
    at net.minecraft.client.render.WorldRenderer.render(WorldRenderer:1240)
    at net.minecraft.client.render.GameRenderer.renderWorld(GameRenderer:717)
    at net.minecraft.client.render.GameRenderer.render(GameRenderer:540)
    at net.minecraft.client.MinecraftClient.render(MinecraftClient:1021)
    at net.minecraft.client.MinecraftClient.run(MinecraftClient:654)
    at net.minecraft.client.main.Main.main(Main:215)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
    at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
    at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Block Entity Details --
  Name: betternether:sign // paulevs.betternether.blockentities.BNSignBlockEntity
  Block: Block{betternether:sign_wart}[floor=false,rotation=4,waterlogged=false]
  Block location: World: (-1232,82,-1213), Chunk: (at 0,5,3 in -77,-76; contains blocks -1232,0,-1216 to -1217,255,-1201), Region: (-3,-3; contains chunks -96,-96 to -65,-65, blocks -1536,0,-1536 to -1025,255,-1025)
  Block: Block{betternether:sign_wart}[floor=false,rotation=4,waterlogged=false]
  Block location: World: (-1232,82,-1213), Chunk: (at 0,5,3 in -77,-76; contains blocks -1232,0,-1216 to -1217,255,-1201), Region: (-3,-3; contains chunks -96,-96 to -65,-65, blocks -1536,0,-1536 to -1025,255,-1025)
  Block Entity NBT: {Color:"black",x:-1232,Text4:'{"text":""}',y:82,Text3:'{"text":""}',z:-1213,Text2:'{"text":""}',id:"betternether:sign",Text1:'{"text":""}'}

-- Affected level --
  All players: 2 total; [class_746['TarmacFFS'/228, l='ClientLevel', x=-1229.26, y=81.00, z=-1212.43], class_745['LittleTarmac'/5373, l='ClientLevel', x=-1186.44, y=91.00, z=-1197.36]]
  Chunk stats: Client Chunk Cache: 5041, 4421
  Level dimension: minecraft:overworld
  Level spawn location: World: (-69,64,112), Chunk: (at 11,4,0 in -5,7; contains blocks -80,0,112 to -65,255,127), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
  Level time: 5740208 game time, 287344 day time
  Server brand: fabric
  Server type: Non-integrated multiplayer server

-- System Details --
  Minecraft Version: 1.16.1
  Minecraft Version ID: 1.16.1
  Operating System: Windows 10 (amd64) version 10.0
  Java Version: 1.8.0_51, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 4056169216 bytes (3868 MB) / 17012097024 bytes (16224 MB) up to 17179869184 bytes (16384 MB)
  CPUs: 16
  JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx16G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
  Suspected Mods: Fabric Loader (fabricloader), Better Nether (betternether)
  Fabric Mods: adabraniummod: Adabranium Mod 1.1.5.1
               adorn: Adorn 1.9.1+1.16.1
               aopalliance_aopalliance: aopalliance 1.0
               autoconfig1u: Auto Config v1 Updated 3.2.0-unstable
               betternether: Better Nether 3.4.3
               blockus: Blockus 1.6.2
               blue_endless_jankson: jankson 1.2.0
               byg: BYG 1.0.9
               campanion: Campanion 1.1.12+build.33
               cardinal-components: Cardinal Components API 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-base: Cardinal Components API (base) 2.4.1
               cardinal-components-block: Cardinal Components API (blocks) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-chunk: Cardinal Components API (chunks) 2.4.1
               cardinal-components-entity: Cardinal Components API (entities) 2.4.1
               cardinal-components-item: Cardinal Components API (items) 2.4.0
               cardinal-components-level: Cardinal Components API (world saves) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-util: Cardinal Components API (utilities) 2.4.0-nightly.1.16-rc1.build.2
               cardinal-components-world: Cardinal Components API (worlds) 2.4.0-nightly.1.16-rc1.build.2
               cavebiomes: Cave Biomes 0.2.3
               chains: Chains Link 1.1.1
               chunkactivator: Chunk Activator 2.1.0+mc1.16.1
               cloth-basic-math: Cloth Basic Math 0.5.1
               cloth-client-events-v0: Cloth Client Events v0 1.0.4
               cloth-config2: Cloth Config v4 4.6.0
               com_fasterxml_jackson_core_jackson-annotations: jackson-annotations 2.9.0
               com_fasterxml_jackson_core_jackson-core: jackson-core 2.9.0
               com_fasterxml_jackson_core_jackson-databind: jackson-databind 2.9.0
               com_fasterxml_jackson_dataformat_jackson-dataformat-yaml: jackson-dataformat-yaml 2.9.0
               com_fasterxml_jackson_module_jackson-module-kotlin: jackson-module-kotlin 2.9.0
               com_google_inject_guice: guice 4.2.3
               com_moandjiezana_toml_toml4j: toml4j 0.7.2
               convth: Convenient Things 0.8.1
               cotton: Cotton 1.0.5
               cotton-cauldron: Cotton Cauldron 1.0.4
               cotton-commons: Cotton Commons 1.0.4
               cotton-config: Cotton Config 1.0.0-rc.7
               cotton-datapack: Cotton Datapack 1.0.5
               cotton-logging: Cotton Logging 1.0.0-rc.4
               cotton-player-events: Cotton Player Events 1.0.2
               couplings: Couplings 1.3.1
               decorative: Decorative 2.0.0 - Summer 2020 Update
               dungeons: JavaDungeons 1.2.1+1.16.1
               economy: FabricEconomy 1.2
               essentials-base: Essentials 0.2.3+1.16.1
               euclid: Euclid 1.1
               exotic-art-core: Exotic Art Core Content Pack 1.0.341
               exotic-matter: Exotic Matter 2.4.343
               fabric: Fabric API 0.15.1+build.380-1.16.1
               fabric-api-base: Fabric API Base 0.1.3+12a8474c7c
               fabric-biomes-v1: Fabric Biomes (v1) 0.2.7+059ea86602
               fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea8902
               fabric-command-api-v1: Fabric Command API (v1) 1.0.8+5ce5339802
               fabric-commands-v0: Fabric Commands (v0) 0.2.0+52d3083602
               fabric-containers-v0: Fabric Containers (v0) 0.1.8+045df74f02
               fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.9+059ea86602
               fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d02
               fabric-diagonal-panes: Fabric Diagonal Panes 0.3.0
               fabric-dimensions-v1: fabric-dimensions-v1 1.0.0+a71b305302
               fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.3.3+7066030f02
               fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b02
               fabric-furnaces: Fabric Furnaces 1.3.1-1.16.1
               fabric-item-api-v1: Fabric Item API (v1) 1.0.0+16acbe5b02
               fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.0+438f963602
               fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+f404f3be02
               fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c502
               fabric-language-kotlin: Fabric Language Kotlin 1.3.72+build.1
               fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.1.0+013e49d202
               fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.0+386eb69e02
               fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce9902
               fabric-models-v0: Fabric Models (v0) 0.1.0+dfdb52d602
               fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.5+b50ffc7b02
               fabric-networking-v0: Fabric Networking (v0) 0.1.10+e00ecb5f02
               fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.5.6+a2d21ddd02
               fabric-object-builders-v0: Fabric Object Builders (v0) 0.6.1+a2d21ddd02
               fabric-particles-v1: fabric-particles-v1 0.2.1+0a6f2a7002
               fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.3.8+7dba2d6c02
               fabric-renderer-api-v1: Fabric Renderer API (v1) 0.2.13+eae12eb802
               fabric-renderer-indigo: Fabric Renderer - Indigo 0.3.3+0a55f37902
               fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.0.1+5a0f9a6002
               fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.3+b7f9825d02
               fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.8+b7084faa02
               fabric-rendering-v0: Fabric Rendering (v0) 1.1.0+5341049002
               fabric-rendering-v1: Fabric Rendering (v1) 1.1.2+346247d702
               fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.2.6+f41e209802
               fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+97324d1102
               fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.2.7+a4c57d8e02
               fabric-textures-v0: Fabric Textures (v0) 1.0.4+eae12eb802
               fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.1.4+5794386e02
               fabricloader: Fabric Loader 0.9.0+build.204
               fermion: Fermion 2.1.191
               fermion-modkeys: Fermion Modifier Keys 1.8.180-snapshot
               fermion-orientation: Fermion Orientation 1.0.180-snapshot
               fermion-varia: Fermion Varia 2.1.191
               gravestones: Gravestones 1.4.0
               hookshot: Hookshot 5.0
               hotm: Heart of the Machine 1.16.1-0.1.2
               identity: Identity 1.4.0-beta-1.16
               illuminations: Illuminations 0.5.1
               inmis: Inmis 1.2.0-1.16.1
               jankson: Jankson 3.0.0+j1.2.0
               javax_inject_javax_inject: javax.inject 1
               kibe: Kibe 1.1.11-BETA
               lambdynlights: LambDynamicLights 1.2.2+1.16.1
               libblockattributes_core: LibBlockAttributes (Core) 0.7.0
               libcd: LibCapableData 2.5.0+1.16.1
               libgui: LibGui 2.1.1+1.16.1
               libmultipart: LibMultiPart 0.4.0
               libmultipart_deps_container: LibMultiPart (Dependency Container) 0.4.0
               libnetworkstack: Lib Network Stack 0.4.0
               libstructure: Libstructure 1.3
               lightblock: Light Block 1.16+1.1.0+build.8
               lightoverlay: Light Overlay 5.4.1
               loadcatcher: Entrypoint Catcher 2.0.0
               magna: Magna 1.1.1-1.16.1
               minecraft: Minecraft 1.16.1
               mooblooms: Mooblooms 1.3.1
               mysql_mysql-connector-java: mysql-connector-java 5.1.46
               no_fog: No Fog 1.0.0-beta.4+1.16.1
               notenoughcrashes: Not Enough Crashes 2.1.0+1.16.1
               org_jetbrains_annotations: annotations 17.0.0
               org_jetbrains_exposed_exposed-core: exposed-core 0.23.1
               org_jetbrains_exposed_exposed-java-time: exposed-java-time 0.23.1
               org_jetbrains_exposed_exposed-jdbc: exposed-jdbc 0.23.1
               org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.3.72
               org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.3.72
               org_jetbrains_kotlinx_kotlinx-coroutines-core: kotlinx-coroutines-core 1.3.7
               org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.3.7
               org_postgresql_postgresql: postgresql 42.2.2
               org_slf4j_slf4j-api: slf4j-api 1.8.0-beta1
               org_xerial_sqlite-jdbc: sqlite-jdbc 3.28.0
               org_yaml_snakeyaml: snakeyaml 1.26
               parchment: Parchment 1.0.2+1.15.2
               phosphor: Phosphor 0.6.0+build.7
               playerabilitylib: Pal 1.2.0
               reach-entity-attributes: Reach Entity Attribute 1.0.1
               rgbblocks: Rgb Blocks 2.1.2
               ringsofascension: Rings of Ascension 1.0.3
               roughlyenoughitems: Roughly Enough Items 4.10.0
               rpgstats: RPGStats 2.1.0+1.16.1
               simpledrawers: Simple Drawers 1.8.0-1.16.1
               sit: Sit 1.16.1-7
               slotlink: slotlink 0.3.6+minecraft-1.16.1
               special-circumstances: Special Circumstances 1.6.191
               spinnery: Spinnery 3.1.12
               spruceui: SpruceUI 1.5.8
               trade: Trade 1.1.0
               trapexpansion: Trap Expansion 1.3.2
               trinkets: Trinkets 2.6.3
               universalcomponents: Universal Components 0.7.2+1.16-pre2
               vanilla-hammers: Vanilla Hammers 1.10.1-1.16.1
               vanilla_parts: Vanilla Parts 1.1.3+1.16.1
               vanillaautomated: Vanilla Automated 1.1.2
               verticalslabs: Vertical Slabs 1.3.0
               voxelmap: VoxelMap 1.10.4
               winged: Winged Mod 1.16-1.2
               woolplates: Wool Plates 1.0.1
               worldedit: WorldEdit 7.2.0-beta-02+d10d7d6
               xb: Exotic Blocks 1.4.142
  Launched Version: fabric-loader-0.9.0+build.204-1.16.1
  Backend library: LWJGL version 3.2.2 build 10
  Backend API: AMD Radeon VII GL version 4.6.13586 Compatibility Profile Context 19.12.2 26.20.15002.61, ATI Technologies Inc.
  GL Caps: Using framebuffer using OpenGL 3.0
  Using VBOs: Yes
  Is Modded: Definitely; Client brand changed to 'fabric'
  Type: Client (map_client.txt)
  Resource Packs: 
  Current Language: English (US)
  CPU: 16x AMD Ryzen 7 2700X Eight-Core Processor
  Client Crashes Since Restart: 1
  Integrated Server Crashes Since Restart: 0

```

**Comment by Greg-J** [comment]
I just replaced that region file with the one from my last backup and loaded the world. Placed the wart sign on a stone wall and crashed again. Seems something is bugged with the wart signs.

**Comment by paulevsGitch** [comment]
I fixed this in development code, fix will be in 3.5.0, I will release it as soon as possible

**Comment by paulevsGitch** [comment]
Fixed in 3.5.0. If this issue will appear again please reopen it
